python/PiFinder/ui/preview.py

```diff
@@ -56,13 +60,6 @@ class UIPreview(UIModule):
             "options": ["Save", "Exit"],
             "callback": "save_exp",
         },
-        "Zoom View": {
```

Comment:
Zoom View is handy when not in self-align mode, removing this implies we'll always be in self-align mode? If it works correctly why not of course. Or is the self-align mode to distinguish between just the preview, and the star-selecting preview?

Reply:
Once this code is merged the RA/DEC will always be reported based on the solve_pixel of the image, which by default will be the center pixel.  The 'B' key is just to enable the star selection feature in the preview, but once the star is selected that pixel becomes the new solve_pixel until it's reset.  The zero key while in star selection mode will reset to center.   Which reminds me I need to do some docs for this :-)